src/Microsoft.VisualStudio.Threading.Tests/SingleThreadedTestSynchronizationContext.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.VisualStudio.Threading.Tests
     using System.Collections.Generic;
     using System.Reflection;
     using System.Threading;
-#if DESKTOP
+#if NETFRAMEWORK
```

Comment:
Shouldn't this be `UseWpfContext`?

```suggestion
#if UseWpfContext
```